experimental/datacommons-js/example-dynamic.html

```diff
@@ -0,0 +1,171 @@
+      rel="stylesheet"
+      href="https://www.datacommons.org/css/ranking.min.css"
+    />
+    <link
+      rel="stylesheet"
+      href="https://www.datacommons.org/css/nl_interface.min.css"
+    />
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.21/lodash.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.0/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/4.6.2/js/bootstrap.min.js"></script>
+    <script src="https://autopush.datacommons.org/datacommons.js"></script>
+    <script>
+      window.onload = function () {
+        // Update web component when text input changes
+        var onTextInputChange = _.debounce((e) => {
+          var $el = $(e.currentTarget);
+          var targetId = $el.data("target-id");
+          var targetAttr = $el.data("target-attr");
+          var $webComponent = $("#" + targetId);
+          $webComponent.attr(targetAttr, $el.val());
+          if (targetAttr === "statVarDcid") {
+            $webComponent.attr("title", $el.val());
+          }
+        }, 400);
+
+        $(".map-control").on("input", onTextInputChange);
+      };
+    </script>
+  </head>
+  <body>
+    <nav class="navbar navbar-expand-lg navbar-light bg-light">
+      <div class="container">
+        <a class="navbar-brand" href="#">Datacommons JS Embedded Charts</a>
+      </div>
+    </nav>
+    <div class="container">
+      <!-- Web component example -->
+      <h2>Map Web Components</h2>
+      <div class="row">
+        <div class="col col-6">
+          <datacommons-map
+            id="map1"
+            title="Count_Person"
+            placeDcid="country/USA"
+            enclosedPlaceType="State"
+            statVarDcid="Count_Person"
+          ></datacommons-map>
+          <form class="mt-3">
+            <div class="form-group">
+              <input
+                type="text"
+                class="form-control map-control"
+                data-target-id="map1"
+                data-target-attr="placeDcid"
+                placeholder="Place DCID"
+                value="country/USA"
+              />
+            </div>
+            <div class="form-group">
+              <input
+                type="text"
+                class="form-control map-control"
+                data-target-id="map1"
+                data-target-attr="enclosedPlaceType"
+                placeholder="Enclosed Place Type (e.g., State, EurostatNUTS1)"
+                value="State"
+              />
+            </div>
+            <div class="form-group">
+              <input
+                type="text"
+                class="form-control map-control"
+                data-target-id="map1"
+                data-target-attr="statVarDcid"
+                placeholder="Statistical Variable DCID"
+                value="Count_Person"
+              />
+            </div>
+          </form>
+        </div>
+        <div class="col col-6">
```

Comment:
just curious what the purpose of putting two example maps with the same controls is?

Reply:
I just like being able to play with two different locations and stat vars side by side 🤷